src/SendGrid/SendGridClient.cs

```diff
@@ -228,6 +228,7 @@ public virtual async Task<Response> MakeRequest(HttpRequestMessage request, Canc
                 RequestUri = new Uri(baseAddress, this.BuildUrl(urlPath, queryParams)),
                 Content = requestBody == null ? null : new StringContent(requestBody, Encoding.UTF8, this.MediaType)
             };
+            request.Content.Headers.ContentType.CharSet = null;
```

Comment:
It looks like this is causing many breaks in the integration tests.

My suggestion would be to implement it as an option with the default being as-is. Ideally, we'd set this only for the relevant endpoints, but I think as a first start, making this configurable is a good step. 

Alternatively, we can simply document how to pass in your own custom client that specifically updates this setting.

Reply:
I believe this needs to be fixed server-side. Sending the CharSet from the client here is perfectly valid. This is a workaround at best. The problem is that the server is rejecting valid requests.

Reply:
See #910 for follow-up regarding the server-side issue.